src/globus_sdk/tokenstorage/v2/base.py

```diff
@@ -20,71 +20,59 @@
 
 class TokenStorage(metaclass=abc.ABCMeta):
     """
-    Abstract base class for interacting with an underlying storage system to manage
-    storage of token data.
+    The interface for interacting with a store of :class:`TokenStorageData` objects.
 
-    The ``namespace`` is a user-supplied way of partitioning data, and any token
-    response data passed to the storage adapter are stored indexed by
-    *resource_server*. If you have a more complex use-case in which this scheme will be
-    insufficient, you should encode that in your choice of ``namespace`` values.
+    Implementations must partition their token data objects by `namespace`.
+    Within a namespace, token data must be indexed by `resource_server`.
```

Comment:
It's definitely not a standard, but the original docstring text matched what I'm accustomed to seeing for docstrings: italics rather than in-line code for parameters.

```suggestion
    Implementations must partition their token data objects by *namespace*.
    Within a namespace, token data must be indexed by *resource_server*.
```

If that's not matching what you want, should these be double-back-ticked?